src/openMVG/sfm/pipelines/sequential/sequential_SfM.cpp

```diff
@@ -1416,7 +1414,12 @@ bool SequentialSfMReconstructionEngine::Resection(const std::size_t viewIndex)
         const Vec2 xJ = _features_provider->feats_per_view.at(J)[track.at(J)].coords().cast<double>();
 
         // test if the track already exists in 3D
-        if (_sfm_data.structure.count(trackId) != 0)
+        bool trackIdExists;
+#pragma omp critical
```

Comment:
Yes, I agree. But currently, this could not happen because it is not possible to have new point to triangulate with 2 valid views. If it's the case, this point would have been already triangulated (as the resectioning itself is done sequentially image per image).
So yes, rewriting it to iterate over tracks would be better and easier for parallelization. But AFAIU, the current fix can me merged safely.